test/tool/resources/slice.json

```diff
@@ -0,0 +1,240 @@
+{
```

Comment:
(I'm skipping over the details in this because I assume it's the same test data as in the other PR, just that this is the output which was used as the input for that test case)